client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -219,72 +226,79 @@ def get_container_items_by_id(self, item_ids):
             for item_id in item_ids
         }
 
-    def get_representation_info_items(self, representation_ids):
+    def get_representation_info_items(self, project_name, representation_ids):
         output = {}
-        missing_repre_ids = set()
+        missing_repre_ids_by_project = {}
+        if project_name is None:
+            project_name = self._controller.get_current_project_name()
```

Comment:
Calling this function should be called in a way that representation ids are pre-grouped by project name. So it should never be allowed to be `None`.  Right?
```suggestion
```